### PR TITLE
Add containers

### PR DESCRIPTION
**Pull request by agrare**
Add container to the ingress_api model

Depends on: ~~https://github.com/ManageIQ/topological_inventory-core/pull/47~~

### CODE REVIEW

**Comment by bdunne** [comment]
Is there an associated ingress_api PR updating the swagger.yaml?

**Comment by agrare** [comment]
@bdunne yes https://github.com/ManageIQ/topological_inventory-ingress_api/pull/6

File: docs/Container.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# TopologicalInventory::IngressApi::Client::Container
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**container_group** | [**InventoryObjectLazy**](InventoryObjectLazy.md) |  | 
+**name** | **String** |  | 
+**cpu_limit** | **Float** |  | [optional] 
+**cpu_request** | **Float** |  | [optional] 
+**memory_limit** | **Float** |  | [optional]
```
Comment (bdunne) [inline docs/Container.md]: Does this need to be regenerated now?

Comment (agrare) [reply docs/Container.md]: :+1: forgot to update the docs dir

**Pull request merged**